### PR TITLE
Fix monitoring database

### PR DESCRIPTION
**Pull request by apelisse**
Fix missing database in creation script and set a limit on influxdb process

---

This change is [Reviewable](https://reviewable.kubernetes.io/reviews/kubernetes/test-infra/757)

### CODE REVIEW

File: velodrome/grafana-stack/influxdb.yaml [hunk 1]
```diff
@@ -22,7 +22,10 @@ spec:
               name: influxdb
               key: rootpassword
         - name: PRE_CREATE_DB
-          value: github
+          value: github;monitoring
```
Comment (spxtr) [inline velodrome/grafana-stack/influxdb.yaml]: Does this mean anything special? Where is this used?

Comment (apelisse) [inline velodrome/grafana-stack/influxdb.yaml]: I'm glad you ask!
Sure of course, it's not just random :-)

https://github.com/tutumcloud/influxdb/blob/master/0.13/run.sh#L100

This variable is used to create the table on the initialization (first run) of the influxdb container.

**Comment by spxtr** [comment]
@apelisse what's the status of this?

**Comment by apelisse** [comment]
We should probably get it merged :-). It's first time setup related so we don't hit it ...

**Comment by spxtr** [comment]
SGTM, LGTM, etc etc

**Comment by apelisse** [comment]
How is manual merging is still a thing? :D

**Pull request merged**